list_view/list_view.h

```diff
@@ -58,6 +58,22 @@ class ListView {
         }
     };
 
+    class ItemTransaction {
```

Comment:
What about the rule of five? The class has a reference to other object as member....

Reply:
Thanks, I've deleted the copy and move operators and constructors.